<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>966</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-6</x>
      <y>-5</y>
      <width>971</width>
      <height>521</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/background/fight_v2.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>-420</x>
      <y>200</y>
      <width>901</width>
      <height>511</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/background/table.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>881</width>
      <height>361</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 110);
 border-radius: 20px;	</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_seleccionar">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>440</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
font: 12pt &quot;Showcard Gothic&quot;;
color: rgb(255, 255, 255);
background-color: rgb(246, 101, 4);</string>
    </property>
    <property name="text">
     <string>Seleccionar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_timer">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>50</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 26pt &quot;Showcard Gothic&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>10</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_contrincante">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 26pt &quot;Showcard Gothic&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>oponente</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_jugador">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>50</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 26pt &quot;Showcard Gothic&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Jugador</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="carta_oponente">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>181</width>
      <height>261</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/cartas/back.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ficha_1">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>20</y>
      <width>55</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/elementos/fichas/agua_verde.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ficha_2">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>20</y>
      <width>55</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/elementos/fichas/fuego_azul.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ficha_3">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>20</y>
      <width>55</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/elementos/fichas/nieve_rojo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ficha_4">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>40</y>
      <width>55</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/elementos/fichas/agua_verde.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ficha_7">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>60</y>
      <width>55</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/elementos/fichas/agua_verde.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="carta_seleccionada">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>100</y>
      <width>181</width>
      <height>261</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/cartas/rojo_nieve_1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="carta_1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>390</y>
      <width>71</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/cartas/azul_agua_1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ficha_5">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>40</y>
      <width>55</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/elementos/fichas/fuego_azul.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ficha_8">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>60</y>
      <width>55</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/elementos/fichas/fuego_azul.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ficha_6">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>40</y>
      <width>55</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/elementos/fichas/nieve_rojo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ficha_9">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>60</y>
      <width>55</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/elementos/fichas/nieve_rojo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ficha_10">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>80</y>
      <width>55</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/elementos/fichas/agua_verde.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ficha_13">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>100</y>
      <width>55</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/elementos/fichas/agua_verde.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ficha_11">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>80</y>
      <width>55</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/elementos/fichas/fuego_azul.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ficha_14">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>100</y>
      <width>55</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/elementos/fichas/fuego_azul.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ficha_12">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>80</y>
      <width>55</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/elementos/fichas/nieve_rojo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ficha_15">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>100</y>
      <width>55</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/elementos/fichas/nieve_rojo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="otro_8">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>55</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/elementos/fichas/fuego_azul.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="otro_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>55</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/elementos/fichas/agua_verde.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="otro_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>55</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/elementos/fichas/nieve_rojo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="otro_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>55</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/elementos/fichas/agua_verde.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="otro_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>55</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/elementos/fichas/agua_verde.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="otro_12">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>55</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/elementos/fichas/nieve_rojo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="otro_6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>55</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/elementos/fichas/nieve_rojo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="otro_15">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>55</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/elementos/fichas/nieve_rojo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="otro_9">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>55</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/elementos/fichas/nieve_rojo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="otro_14">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>55</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/elementos/fichas/fuego_azul.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="otro_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>55</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/elementos/fichas/agua_verde.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="otro_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>55</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/elementos/fichas/agua_verde.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="otro_11">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>55</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/elementos/fichas/fuego_azul.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="otro_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>55</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/elementos/fichas/fuego_azul.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="otro_5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>55</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/elementos/fichas/fuego_azul.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>941</width>
      <height>421</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/background/table.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="carta_oponente_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>31</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/cartas/back.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="carta_oponente_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>31</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/cartas/back.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="carta_oponente_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>300</y>
      <width>31</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/cartas/back.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="carta_oponente_5">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>200</y>
      <width>31</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/cartas/back.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="carta_oponente_6">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>260</y>
      <width>31</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/cartas/back.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="carta_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>390</y>
      <width>71</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/cartas/azul_agua_1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="carta_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>390</y>
      <width>71</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/cartas/azul_agua_1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="carta_4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>390</y>
      <width>71</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/cartas/azul_agua_1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="carta_5">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>390</y>
      <width>71</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/cartas/azul_agua_1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>label_3</zorder>
   <zorder>label_2</zorder>
   <zorder>ficha_1</zorder>
   <zorder>ficha_2</zorder>
   <zorder>ficha_3</zorder>
   <zorder>ficha_4</zorder>
   <zorder>ficha_7</zorder>
   <zorder>ficha_5</zorder>
   <zorder>ficha_8</zorder>
   <zorder>ficha_6</zorder>
   <zorder>ficha_9</zorder>
   <zorder>ficha_10</zorder>
   <zorder>ficha_13</zorder>
   <zorder>ficha_11</zorder>
   <zorder>ficha_14</zorder>
   <zorder>ficha_12</zorder>
   <zorder>ficha_15</zorder>
   <zorder>otro_1</zorder>
   <zorder>otro_3</zorder>
   <zorder>otro_6</zorder>
   <zorder>otro_9</zorder>
   <zorder>otro_4</zorder>
   <zorder>otro_2</zorder>
   <zorder>otro_5</zorder>
   <zorder>otro_7</zorder>
   <zorder>otro_10</zorder>
   <zorder>otro_13</zorder>
   <zorder>otro_8</zorder>
   <zorder>otro_11</zorder>
   <zorder>otro_14</zorder>
   <zorder>otro_12</zorder>
   <zorder>otro_15</zorder>
   <zorder>label_4</zorder>
   <zorder>carta_1</zorder>
   <zorder>boton_seleccionar</zorder>
   <zorder>carta_oponente</zorder>
   <zorder>label_contrincante</zorder>
   <zorder>label_timer</zorder>
   <zorder>label_jugador</zorder>
   <zorder>carta_seleccionada</zorder>
   <zorder>carta_oponente_2</zorder>
   <zorder>carta_oponente_3</zorder>
   <zorder>carta_oponente_4</zorder>
   <zorder>carta_oponente_5</zorder>
   <zorder>carta_oponente_6</zorder>
   <zorder>carta_2</zorder>
   <zorder>carta_3</zorder>
   <zorder>carta_4</zorder>
   <zorder>carta_5</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
